src/System.IO.Pipelines/src/System/IO/Pipelines/PipeReader.cs

```diff
@@ -53,10 +53,20 @@ public abstract partial class PipeReader
         /// <summary>
         /// Returns a <see cref="Stream"/> that wraps the <see cref="PipeReader"/>.
```

Comment:
[Borrowing from StreamReader](https://github.com/dotnet/dotnet-api-docs/blob/85b7af914b69213a8e7ef7febacf2f9a5a99a13e/xml/System.IO/StreamReader.xml#L917)

```suggestion
         /// Returns a <see cref="Stream"/> that wraps the <see cref="PipeReader"/>, which optionally leaves the pipe open when disposed.
```